src/parser/transform/expression/transform_function.cpp

```diff
@@ -39,6 +40,36 @@ static ExpressionType WindowToExpressionType(string &fun_name) {
 	return ExpressionType::WINDOW_AGGREGATE;
 }
 
+static string EncodeOrderType(OrderType type) {
+	switch (type) {
+	case OrderType::INVALID:
```

Comment:
Perhaps this can use `EnumSerializer::EnumToString` instead?

Reply:
Unfortunately the encoder doesn't produce the strings that `list_sort` expects (e.g., "ASCENDING" vs "ASC")!

Reply:
Could we perhaps fix that, e.g. by expanding what list_sort expects (or switching that over to the EnumSerializer as well)?